experimenter/experimenter/experiments/models.py

```diff
@@ -793,33 +793,54 @@ def computed_duration_days(self):
         return self.proposed_duration
 
     def timeline(self):
-        return [
+        timeline_entries = [
             {
                 "label": self.Status.DRAFT,
                 "date": self.draft_date,
                 "is_active": self.is_draft,
+                "days": None,
             },
             {
                 "label": self.Status.PREVIEW,
                 "date": self.preview_date,
                 "is_active": self.is_preview,
+                "days": None,
             },
             {
                 "label": self.PublishStatus.REVIEW,
                 "date": self.review_date,
                 "is_active": self.is_review,
+                "days": None,
             },
             {
                 "label": self.Status.LIVE,
                 "date": self.start_date,
                 "is_active": self.is_live,
+                "days": None,
             },
             {
                 "label": self.Status.COMPLETE,
                 "date": self.computed_end_date,
                 "is_active": self.is_complete,
+                "days": self.computed_duration_days if self.computed_end_date else None,
```

Comment:
Related to my other comment about dates -- why do we use the computed duration here, but the actual enrollment for that one?

It also just occurred to me that the answer to all of my questions might be "because this is how we have it on the current pages", which is a perfectly valid answer!

Reply:
yup I referred to the current pages and translated that

Reply:
Ok fair enough. Still seems a bit strange to me, but maybe this was discussed and agreed upon already?

Reply:
if its not ended we use computed one. but if its ended computed one returns the actual one